Codegen/src/Nodes/TypedDataNode.cs

```diff
@@ -972,11 +972,15 @@ private static System.Text.StringBuilder GetIndent(int level, int indent, bool i
 
         };
 
-        public static List<ITextLine> InsertChildren(ColumnsLayout? layout, List<string> rootProcedures, List< Tuple<string,string> > rootVariableName, DataDefinition ownerDefinition, DataDefinition type, int level, int indent)
+        public static List<ITextLine> InsertChildren(ColumnsLayout? layout, List<string> rootProcedures, List<Tuple<string, string>> rootVariableName, DataDefinition ownerDefinition, DataDefinition type, int level, int indent)
+        {
+            return InsertChildren(layout, rootProcedures, rootVariableName, ownerDefinition, type, level, indent, null);
+        }
+
+        private static List<ITextLine> InsertChildren(ColumnsLayout? layout, List<string> rootProcedures, List< Tuple<string,string> > rootVariableName, DataDefinition ownerDefinition, DataDefinition type, int level, int indent, CopyDirective usedCopy)
```

Comment:
So here you add a new `CopyDirective usedCopy` parameter to track the current COPY instruction and to know whether it should be written or not. But @smedilol confirmed that COPY instructions may be found in copybook content so could you please add a test having a COPY within a COPY ? I think we may need a `Stack<CopyDirective>` here.

Reply:
- Test completed with adding use of copy of copy (CopyInsideTypeDefInsideDependency).
- In method InsertChildren, when a copy is found, the parsing of nodes is interrompted by instruction "continue" (line 1013); the copy below the copy is not ultimately considered.
https://github.com/TypeCobolTeam/TypeCobol/pull/1646/commits/5ec741f218377464909aae42e8454ea4de4d1d2e